### DISCUSSION TITLE
Welcome to the FlexDeploy Community on GitHub

### DISCUSSION THREAD

**Discussion by erickjones2** [FlexDeploy-Community/community#4] [Announcements]
We are excited to announce the FlexDeploy Community is now on GitHub!

- Engage with other FlexDeploy users to collaborate, ask questions, advance your adoption, and drive continuous improvement in your DevOps Journey
- Submit product ideas to vet within the Community
- Engage with Flexagon resources - FlexDeploy Product Management and Engineering teams 
- Contribute ideas, plugins, workflows, solutions, code examples

Please be advised that the community discussions are not to be used in place of Flexagon Support to report issues or ask time sensitive questions.  The discussions are for collaboration within the community.  While Flexagon personnel may engage in discussions, responses are not bound to an SLA.  All discussions are also visible across the entire community.

**Comment by GH-Coby-Wan** [comment]
Is there a location where your customers share different integrations? Such as groovy scripts that they've created for incoming or outgoing webhooks. I'm looking for a way to update the package status to 'Completed' once a package has been successfully delivered to production.

**Reply by erickjones2** [reply]
Hi Coby,

Yes, you can contribute scripts and other artifacts to the community repository.  See the readme on the repository itself.
https://github.com/FlexDeploy-Community/community

After you take a look, let us know if you have questions.

Thanks,
Erick
